src/openfl/display/BitmapData.hx

```diff
@@ -916,6 +927,11 @@ class BitmapData implements IBitmapDrawable {
 			
 			// __bufferAlpha = alpha;
 			// __bufferColorTransform = colorTransform != null ? colorTransform.__clone () : null;
+			
```

Comment:
Perhaps this might make sense, although this case is only run when `__buffer` is `null` already or if the GL context was restored, in which case your buffer object is gone anyway, so I think we could skip it

Reply:
Hmm, I guess you're right...